### PR TITLE
[EN] Beatmapping: Rewrite

### PR DESCRIPTION
**Pull request by MegaApplePi**
- rewrite parts of the page
- remove copypasta and redirect the reader to other pages
- ~~steal~~ reset from @EpicSupermind's version in #352

### CODE REVIEW

File: wiki/Beatmapping/en.md [hunk 1]
```diff
@@ -1,337 +1,56 @@
...
-How do I start a new beatmap?
------------------------------
+### Timing
 
-As osu! is a rhythm based game, the songs you choose to map should have an obvious and constant beat. Try to avoid songs which have tempo (speed) changes or slow songs until you are more experienced and used to the beatmapping mechanism.
+_See also: [Timing](/wiki/Timing)._
 
-To start a new beatmap, follow these simple steps:-
+**Timing is vital!**
+An incorrectly timed maps will not be considered for ranking.
```
Comment (EpicSupermind) [inline wiki/Beatmapping/en.md]: An incorrectly timed maps?
`Incorrectly timed maps will not be considered for ranking.`

File: wiki/Beatmapping/en.md [hunk 2]
```diff
@@ -1,337 +1,56 @@
...
+### Beatmap Editor
 
--   It should open and direct you to the edit screen with your audio file at there.
--   Your new song will be at the bottom of the song list, highlighted in purple.
+_See also: [Beatmap Editor](/wiki/Beatmap_Editor)._
 
--   Click on the song to open up Song Setup.
-    -   Insert and check that the song title and artist are correct, and add Source and Tags if necessary, then enter in an appropriate difficulty name (Easy, Insane and Cruisin' are examples, you can change it later anyway).
--   You can set the other options now if you want, but they can be changed later on. Click OK, and you can start beatmapping! The next step is to add timing to your beatmap.
+The beatmap editor is where you will probably spend, if not, a majority of your time here in the beatmap editor.
```
Comment (EpicSupermind) [inline wiki/Beatmapping/en.md]: Rephrase I guess.
`The beatmap editor is where you will probably spend most, if not alll of your time beatmapping.`

Comment (MegaApplePi) [inline wiki/Beatmapping/en.md]: whoops, I think this was meant to talk about the Compose tab not the beatmap editor

File: wiki/Beatmapping/en.md [hunk 3]
```diff
@@ -1,337 +1,56 @@
...
+## Getting Started
 
--   [Easy](Easy_(Difficulty) "wikilink")
--   [Normal](Normal "wikilink")
--   [Hard](Hard "wikilink")
--   [Insane](Insane "wikilink")
--   [Expert](Expert "wikilink")
+To make a new beatmap, go to the edit mode then drag-and-drop an audio file into _osu!_.
+(Alternatively, you can move an audio file onto the `Songs` folder.)
+This should then open and direct you to the edit screen with your audio file.
```
Comment (TPGPL) [inline wiki/Beatmapping/en.md]: this `then` seems 👎

File: wiki/Beatmapping/en.md [hunk 4]
```diff
@@ -1,337 +1,56 @@
...
+Your new song will be at the bottom of the song list, highlighted in purple.
```
Comment (TPGPL) [inline wiki/Beatmapping/en.md]: will be available?

File: wiki/Beatmapping/en.md [hunk 5]
```diff
@@ -1,337 +1,56 @@
...
+If it is not listed here for any reason, press `F5` to reset the beatmap cache and it should appear.
```
Comment (TPGPL) [inline wiki/Beatmapping/en.md]: here -> there

File: wiki/Beatmapping/en.md [hunk 6]
```diff
@@ -1,337 +1,56 @@
...
+### Song Setup
 
-It is also not uncommon for [mappers](Mappers "wikilink") to use custom [difficulty](Difficulty "wikilink") names in place of these. (Like "Mario" for example)
+_See also: [Song Setup](/wiki/Song_Setup)_
 
-Additionally you may find [Guest Difficulties](Guest_Difficulty "wikilink") which are commonly labeled as mapper's difficulty (For example: "Larto's Insane").
+When making a new [mapset](/wiki/mapset), you will always see this dialog before any other parts of the beatmap editor.
 
-With the addition of the Special modes, you can now add mod-specific difficulties to your beatmap other than the usual osu! difficulty. It is entirely possible to fill the beatmap with only Taiko/Catch the Beat/Osu!mania difficulties provided there is an Easy/Normal difficulty in it.
+You should fill out this information before you create another [difficulty](/wiki/difficulty).
```
Comment (TPGPL) [inline wiki/Beatmapping/en.md]: Before creating

File: wiki/Beatmapping/en.md [hunk 7]
```diff
@@ -1,337 +1,56 @@
...
 
--   Click on the song to open up Song Setup.
-    -   Insert and check that the song title and artist are correct, and add Source and Tags if necessary, then enter in an appropriate difficulty name (Easy, Insane and Cruisin' are examples, you can change it later anyway).
--   You can set the other options now if you want, but they can be changed later on. Click OK, and you can start beatmapping! The next step is to add timing to your beatmap.
+The compose tab of the beatmap editor is where you will probably spend a majority of your time beatmapping.
+It is where you can visually place hit objects and toggle their [hit sounds](/wiki/hit_sounds)
 
-How do I time a song properly?
-------------------------------
+### Storyboards
```
Comment (TPGPL) [inline wiki/Beatmapping/en.md]: expand this header a bit

File: wiki/Beatmapping/en.md (hunk 3 above)
Comment (oralekin) [inline wiki/Beatmapping/en.md]: It does sound kinda odd but I believe it is used correctly.

File: wiki/Beatmapping/en.md (hunk 7 above)
Comment (MegaApplePi) [inline wiki/Beatmapping/en.md]: to ...?

Comment (TPGPL) [inline wiki/Beatmapping/en.md]: It feels too short compared to the others

Comment (TPGPL) [inline wiki/Beatmapping/en.md]: Maybe add an information what a storyboard is?

**Pull request merged**